assets/queries/k8s/anonymous_auth_is_not_set_to_false/query.rego

```diff
@@ -39,6 +39,42 @@ CxPolicy[result] {
 	}
 }
 
+CxPolicy[result] {
+	resource := input.document[i]
+	specInfo := k8sLib.getSpecInfo(resource)
+	container := specInfo.spec.containers[j]
+
+	inArray(container.command, "kubelet")
+	containerArgs := object.get(container, "args", {})
+	inArray(containerArgs, "--anonymous-auth=true")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("spec.container[%d].args", [j]),
```

Comment:
If you only specify `spec.container[...]`, without `"metadata.name={{%s}}` in front, searchKey will potentially point to the wrong `document`, if a manifest includes multiple resources with containers that have the same name (or index in your case)